lib/util/move-file.js

```diff
@@ -1,9 +1,12 @@
 'use strict'
 
-var fs = require('graceful-fs')
+const fs = require('graceful-fs')
+const Promise = require('bluebird')
```

Comment:
I prefer to call my bluebird class `Bluebird` because it makes it immediately clear it doesn't have the same interface.